views/layout.php

```diff
@@ -50,36 +47,40 @@
         
         <body>
             <!-- Mise en place de la topbar -->
-            <?php
-                include 'views/'.$topbar.'.php';
+                <?php
+                include ($topbar);            
             ?>
 
             <!-- Mise en place du contenu -->
             <div class="container">
                 <?php
-                    include 'views/'.$page.'.php';
+                    echo $contentPage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$contentPage'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)